src/components/widgets/EditableList.vue

```diff
@@ -165,7 +177,7 @@ function commitEdit(index: number) {
       >
         Save Changes to {{ entityNamePlural }}
       </button>
-      <button type="button" class="btn btn-secondary" @click.prevent="emit('item:new')">
+      <button type="button" class="btn btn-secondary" @click.prevent="addItem()">
```

Comment:
if `type="button"` is present, does that cover the use case for `.prevent`?

Reply:
Yeap, it does, and has been corrected to use "type button" instead. 💯